.github/ISSUE_TEMPLATE/new-ontology.yml

```diff
@@ -0,0 +1,249 @@
+      options:
+        - label: I have read and understood [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md)
+          required: true
+        - label: There is no other ontology in the OBO Foundry which would be an appropriate place for my terms. If there were, I have contacted the editors, and we decided in mutual agreement that a separate ontology is more appropriate.
+          required: false
+        - label: My ontology has a specific release file with a version IRI and a `dc:license` annotation, serialised in RDF/XML.
+          required: false
+        - label: I understand that term definitions, while not mandatory, are key to understanding the intentions of a term especially when the ontology is used in curation. I made sure that a reasonable majority of terms in my ontology have definitions, using the [IAO:0000115](http://www.ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115) property.
+          required: false
+        - label: For every term in my ontology, I checked whether another OBO Foundry ontology has one with the same meaning. If so, I re-used that term directly (not by cross-reference, by directly using the IRI).
+          required: false
+        - label: For all relationship properties (Object and Data Property), I checked whether [the Relaton Ontology (RO)](http://www.ontobee.org/ontology/catalog/RO?iri=http://www.w3.org/2002/07/owl%23ObjectProperty) includes an appropriate one. I understand that aligning with RO is an essential part of the overall alignment between OBO ontologies!
+          required: false
+        - label: For the selection of appropriate annotation properties, I looked at [OMO](http://www.ontobee.org/ontology/catalog/OMO?iri=http://www.w3.org/2002/07/owl%23AnnotationProperty) first. I understand that aligning ontology metadata and term-level metadata is essential for cross-integration of OBO ontologies.
+          required: false
+        - label: If I was not sure about the meaning of any of the checkboxes above, I have consulted with a member of the OBO Foundry for advice.
+          required: false
+        - label: The requested ID space does not conflict with another ID space found in other registries such as the Bioregistry and BioPortal, see [here](https://obofoundry.org/id-policy) for a complete list.
+          required: false
```

Comment:
should be required.

Reply:
I know all of these should be required, but I think it might just motivate people to check them without _really_ considering them so they can submit the issue

Reply:
Ok fair enough.. Whatever you think is best!

Reply:
Or is there a subset of them that you think should be required before even allowing an issue to be submitted? What policy should we have if people incorrectly check boxes? For example, I think we should close issues if people falsely assert some of these, invite them to open a new issue after they have addressed them, and potentially slow-track future submissions from the submitter

Reply:
I think so too, but I honestly belief they should all be checked. I am happy for the issue to be submitted, but we wont assign a reviewer unless all checkboxes are checked..

Reply:
One thing that confused me a bit is that you should check the boxes saying if there was an issue that you'd get in touch with an OBO member, but it's not really clear how to do this. Like if I didn't understand these and needed support, I'd want to submit the issue then ask for help on it since that would keep the discussion tied to the actual thing that needed fixing

Reply:
Yes, good point. Since the person submitting this PR is already on [obo-discuss](https://groups.google.com/g/obo-discuss), they should send an email there if they need support!

Reply:
Lets do it as a second step... Thanks!